dlp/api/Snippets/DeidentifyWithDateShift.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_deidentify_date_shift]
+
+using Google.Cloud.Dlp.V2;
+using Google.Protobuf;
+using System;
+using System.IO;
+using System.Linq;
+
+public class DeidentifyWithDateShift
+{
+    public static DeidentifyContentResponse Deidentify(
+        string projectId,
+        string inputCsvFilePath,
+        int lowerBoundDays,
+        int upperBoundDays,
+        string dateFields,
+        string contextField = "",
+        string keyName = "",
+        string wrappedKey = "")
+    {
+        var dlp = DlpServiceClient.Create();
+
+        // Read file
```

Comment:
Is it an explicit objective of this sample to show how to read and parse values from a CSV? Could we receive a list of dates and values instead so the users are not lost in all the CSV related code and can focus on DLP? I believe this to be left over from command line running as well. At least consider moving the CSV processing code to a separate method.

Reply:
I'm pretty much copying everything over wholesale and intend to do a pass to make things better, as well. Right now I'm just focused on getting the samples ported over.